pegasus/sites.v3/code.org/views/stats_carousel.haml

```diff
@@ -55,15 +55,17 @@
     });
 
     function update_share_buttons() {
-      var pos = $(id + ' .slides').triggerHandler("currentPosition");
-      if(statistics[pos] != null) {
-        var facebook_anchor = $('#stats-section .share-button-facebook').parents('a.window-popup');
-        var new_facebook_href = facebook_anchor.attr('href');
+      if ($('.share_buttons').length) {
```

Comment:
This problem is interesting enough that maybe a comment would be good to describe why we do this check.

Reply:
✅